### PR TITLE
Added python solutions for Day 4

### PR DESCRIPTION
**Pull request by vishalshirke7**
Please make sure you have done the following:

- [x] Created a folder with the name of language you code inside `/Day<n>/` directory (Where n is for nth day)
- [x] Added your Codes
- [x] Tested your codes
- [x] Added credits (author and date) at the top of the code
- [x] Updated README.md
- [x] Added yourself as a contributor using `all-conributors-cli`

### CODE REVIEW

**Comment by vishalshirke7** [comment]
Hi @MadhavBahlMD  I have resolved the conflicts. Please review the PR

**Comment by MadhavBahlMD** [comment]
Conflicts still exist in the CONTRIBUTORS.md, an easy way would be to take a pull from this repository's remote to your local remote and then run `all-contributors-cli` script once again.

Thanks

**Comment by vishalshirke7** [comment]
Hi @MadhavBahlMD 
I was having issues with adding name in contributors file. seems like the project name and owner have been changed in .all-contributors.src. Anyway I have raised an issue as you said on whatsapp group.

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![All Contributors](https://img.shields.io/badge/all_contributors-14-orange.svg?style=flat-square)](#contributors)
```
Comment (MadhavBahlMD) [inline CONTRIBUTORS.md]: image


Why have you changed number of contributors from 14 to 1?

**Pull request merged**